dev/io.openliberty.cdi.4.1.internal_fat/.classpath

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
```

Comment:
small thing, but I don't think is needed? (at least the other classpaths don't have it)

Reply:
I'll have a check, I think that got added automatically when I switched the java version to 17.

I did have to take a similar attribute out from the BND container since it didn't like that multiple dependencies were providing the same package.

Reply:
It looks like many of our Java 17 fats do have this attribute. It makes the JRE System Library show up under "module path" rather than "classpath" - I would _guess_ that maybe blocks access to classes in certain modules which would otherwise get exposed?

Given that it's present in many of our other Java 17 fats, I'm going to leave it in for now.